lib/client/signalflow/request_manager.js

```diff
@@ -390,11 +390,48 @@ function RequestManager(options) {
     //destroy all open SSE connections
   }
 
+
+  function registerLiveTail(params, callbacks, overrideRequestId){
+    var requestId = overrideRequestId || getRequestIdAndIncrement();
+    knownComputations[requestId] = {
+      onMessage: callbacks.onMessage,
+      transport: transports.WEBSOCKET,
+      requestId: requestId,
+      streamController: {
+        retry: function () {
+          registerLiveTail(params, callbacks, overrideRequestId);
+        },
+        stop: function () {
+          stopLiveTail(requestId);
+        }
+      }
+    };
+    activeSocket.send(JSON.stringify({
+      type: 'registerlivetail',
+      channel: requestId,
+      query: params.query,
+      throttleOptions: params.throttleOptions
+    }));
+  }
+
+  function stopLiveTail(requestId) {
+    if (knownComputations[requestId]) {
```

Comment:
It is doing this 
`      knownComputations[requestId].stopRequested = true;
`

Not sure if I need that. I am not setting stopRequested when I add my computation, so this might error

Reply:
It won't, and it'll be harmless.

Reply:
done